api/app.go

```diff
@@ -367,7 +367,7 @@ func (app *App) configureApplication() {
 	a.Get("/games/:gameID/players/:playerPublicID", RetrievePlayerHandler(app))
 
 	// Clan Routes
-	a.Get("/games/:gameID/clan-search", SearchClansHandler(app))
+	a.Get("/games/:gameID/clans/search", SearchClansHandler(app))
```

Comment:
Any special reason to change this path? It might break compatibility if anyone uses this route